packages/theme-toolkit/src/component-stories-utrecht.tsx

```diff
@@ -470,6 +470,43 @@ export const UTRECHT_COMPONENT_STORIES: ComponentStory[] = [
         </Paragraph>
       </Alert>
     ),
+    detectTokens: {
+      anyOf: [
+        'utrecht.alert.color',
+        'utrecht.alert.background-color',
+        'utrecht.alert.border-color',
+        'utrecht.alert.border-width',
+        'utrecht.alert.border-radius',
+        'utrecht.alert.column-gap',
+        'utrecht.alert.padding-block-start',
+        'utrecht.alert.padding-block-end',
+        'utrecht.alert.padding-inline-start',
+        'utrecht.alert.padding-inline-end',
+        'utrecht.alert.margin-block-start',
+        'utrecht.alert.margin-block-end',
+        'utrecht.alert.message.row-gap',
```

Comment:
It would be convenient to include the icon tokens here.

Reply:
<img width="350" alt="Screenshot 2024-12-24 at 10 31 07" src="https://github.com/user-attachments/assets/30ad32c8-46e4-44db-b3c5-b470108fe49c" />

You mean to add icon token to the right place like the icon (color, size) it self to de default alert, de info to de Alert info etc? 

I just thought its so large that might as well add a new `Alert icon` story, you say it does not exist. How can I see that?